Xamarin.Forms.Xaml/MarkupExtensions/AppThemeBindingExtension.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Globalization;
+using System.Reflection;
+
+namespace Xamarin.Forms.Xaml
+{
+	[ContentProperty(nameof(Default))]
+	public class AppThemeBindingExtension : IMarkupExtension<BindingBase>
+	{
+		public AppThemeBindingExtension()
+		{
+			ExperimentalFlags.VerifyFlagEnabled(nameof(AppThemeBindingExtension), ExperimentalFlags.AppThemeExperimental, nameof(AppThemeBindingExtension));
+		}
+
+		public object Default { get; set; }
+		public object Light { get; set; }
+		public object Dark { get; set; }
+		public object Value	{ get; private set;	}
+
+		public object ProvideValue(IServiceProvider serviceProvider) => (this as IMarkupExtension<BindingBase>).ProvideValue(serviceProvider);
+
+		BindingBase IMarkupExtension<BindingBase>.ProvideValue(IServiceProvider serviceProvider)
```

Comment:
drop the converter as it was only working if the object passed was a Xaml primitive type